src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -192,8 +188,6 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
 
         String body = format(
             CONFIRMATION_SUMMARY,
-            format("/cases/case-details/%s#CaseDocuments", caseData.getCcdCaseReference()),
-            claimIssueConfiguration.getResponsePackLink(),
```

Comment:
`ClaimIssueConfiguration` class is not used anymore, so could be removed (also remove `unspecified.response-pack-url` from config and callback handler test.

Reply:
Should I remove just usages or should I just delete the class as well?

Reply:
This was the only place where configuration class was used, so let's remove it